_src/static/sass/main.scss

```diff
@@ -0,0 +1,66 @@
+// ------------------------------
+// edX [application-name]: Main Style Compile
+
+// About: Sass compile for the edX [application-name]. This does not contain styles for other edX products/experiences (e.g. account/onboarding).
+
+// ------------------------------
+// #VENDOR
+// ------------------------------
+@import '../../vendor/bourbon/bourbon';
+@import '../../vendor/neat/neat';
+// @import '../../vendor/jquery-ui';
```

Comment:
If we're not using this now, my recommendation would be to remove it. We can add it back when we 1) get to more complex emails or 2) need to review our CSS in a setting that also loads JQUI.